lib/joke.rb

```diff
@@ -0,0 +1,13 @@
+class Joke
+attr_reader :joke, :setup, :punchline
+
+  def initialize(joke, setup, punchline)
+    @joke = joke
+    @setup = setup
+    @punchline = punchline
+  end
+
+  def id
+    @joke = 1
```

Comment:
Whenever you find yourself hard-coding something like this, then you can assume there is probably a better way. Hard-coding makes our program less dynamic and more brittle. You were really close to what we were looking for here. We wanted the `joke` attribute you defined to be the `id` attribute, based on the interaction pattern.